modules/apps/27-interchain-accounts/types/validate.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"regexp"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// DefaultMaxAddrLength defines the default maximum character length used in validation of addresses
+var DefaultMaxAddrLength = 64
+
+// DefaultMinAddrLength defines the default minimum character length used in validation of addresses
+var DefaultMinAddrLength = 32
+
+// IsValidAddr defines a regular expression to check if the provided string consists of
+// strictly alphanumeric characters
+var IsValidAddr = regexp.MustCompile("^[a-zA-Z0-9]*$").MatchString
+
+// ValidateVersion performs basic validation of the provided interchainaccounts version string
+func ValidateVersion(version string) error {
```

Comment:
Added the following `ValidateVersion` and tried to maintain flexibility so it can be used for version strings in all handshake callbacks. i.e. supporting version strings where the address is *and* isn't included

Reply:
LGTM :+1: